tests/FSharp.Compiler.ComponentTests/Conformance/Types/StructTypes/StructActivePatterns.fs

```diff
@@ -38,21 +38,21 @@ let rec (|IsOne|_|) someNumber =
 but here has type
     'int option'    """)
 
-    [<Fact>]
```

Comment:
Please resolve this before merging, we should not keep commented out around (or at least add a comment why it is commented out if it is to be resolved shortly)

Reply:
As an idea, since this is a language feature - you could condition the test to be run with a specific language version to make sure it behaves the same for older language versions.

Reply:
Uncommented the test.